### PR TITLE
Brush up Starlit Sentinel Dedication and shinning starlight attack spell

### PR DESCRIPTION
**Pull request by ammalagonc**
- Adds choice set for zodiac constellation
- Item alteration for the spell

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -4719,6 +4719,62 @@
                     "WolfJaws": "Wolf Jaws"
                 }
             },
+            "StarlitSentinel": {
+                "Archer": {
+                    "Description": "You brandish your sentinel weapon, summoning the image of the Archer constellation above your head. The Archer then releases a hail of silver arrows that deals @Damage[(2+(@item.level-1))d10[piercing]] damage to all enemies in a @Template[type:cone|distance:15], with a @Check[reflex|against:spell|basic] save against your arcane spell DC. Enemies that critically fail are @UUID[Compendium.pf2e.conditionitems.Item.TkIyaNPgTZFBCCuh]{Dazzled} until the start of your next turn.",
```
Comment (pedrogrullada) [inline static/lang/re-en.json:4724]: This DC will be wrong for any caster whose casting attribute is not charisma

Comment (ammalagonc) [reply static/lang/re-en.json:4724]: What would you suggest?

Comment (pedrogrullada) [reply static/lang/re-en.json:4724]: I would deal with the damage via Damage Alteration, use variants for the spell save, and description override for the two template shapes

Comment (ammalagonc) [reply static/lang/re-en.json:4724]: image

Wanted to validate what you are saying and noticed when I posted the checks the DC's match with what's on the character sheet. Is there something I'm missing?

**Comment by ammalagonc** [comment]
Closing this one per our conversation.

**Pull request closed without merging**